iotedgedev/__init__.py

```diff
@@ -1,8 +1,6 @@
-# -*- coding: utf-8 -*-
-
 """Top-level package for Azure IoT Edge Dev Tool."""
 
-__author__ = 'Microsoft Corporation'
-__email__ = 'vsciet@microsoft.com'
-__version__ = '2.2.0'
+__author__ = 'Microsoft Open Source'
+__email__ = 'opencode@microsoft.com'
```

Comment:
Is it a standard to use this alias?  Who monitors it?  The old alias was the team, not general alias.

Reply:
This is how the sdk teams do it. I don't believe we have a team email. The emails get filtered down to the team owning the repo.